tabs.js

```diff
@@ -0,0 +1,21 @@
+function openPlayer(cityName) {
+    var i;
+    var x = document.getElementsByClassName("player_tab");
+    for (i = 0; i < x.length; i++) {
+      x[i].style.display = "none";
+    }
+    document.getElementById(cityName).style.display = "block";
+}
```

Comment:
Let's use a more easy to understand variable name:

```suggestion
function openPlayer(playerName) {
    var i;
    var x = document.getElementsByClassName("player_tab");
    for (i = 0; i < x.length; i++) {
      x[i].style.display = "none";
    }
    document.getElementById(playerName).style.display = "block";
}
```